alpine-nix-rails-nginx/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN nix-env --install \
     libxslt-1.1.28 \
     nginx-1.9.4 \
     nodejs-0.12.7 \
+    postgresql-9.4.4 \
```

Comment:
Is there the option to only install the client libs? This would remove the server binary from being available inside the image.